client/components/adminProducts.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import ManageableProduct from './manageableProduct'
+import {fetchProducts, deleteProductById} from '../store/product'
+//TODO: show conditionally in navbar
+class AdminProducts extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {}
+  }
+
+  componentDidMount() {
+    this.props.fetchProducts()
+  }
+
+  deleteProduct = id => {
+    this.props.deleteProduct(id)
+  }
```

Comment:
Why do you have this method? You can just use `this.props.deleteProduct` directly in your `render` method.